src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -500,6 +504,10 @@ $(function () {
           });
           var container = $('#affix > ul');
           container.scrollTop(container.scrollTop() + top - 100);
+
+          // calculate height of #affix
+          var parentHeight = $("#affix").parent().height();
+          $("#affix").css("height", parentHeight - contributionHeight - 50); // minus the height of margin top
         }
```

Comment:
Found the root cause, the `position` of `.affix` should be relative since its parent is fixed.

---
In reply to: [75646927](https://github.com/dotnet/docfx/pull/626#discussion_r75646927) [](ancestors = 75646927,75638785)